helios-services/src/main/java/com/spotify/helios/agent/AddExtraHostContainerDecorator.java

```diff
@@ -17,6 +17,7 @@
 
 package com.spotify.helios.agent;
 
+import com.google.common.base.Optional;
```

Comment:
+1 to Rohan, I think it makes more sense to use java8 optional once it can be used in all places in the codebase.